reference/audit_logs.rst

```diff
@@ -0,0 +1,241 @@
+the lifetime of that websocket connection and each request/response pair will have the same `message-id`, which itself will
+increment whenever a new request is made. Then observing the `is-response` (boolean) and `errors` fields, one can ascertain whether 
+a call was successful.
+
+An example of a request and response audit log are below.
+The first is a request for a model status and the second is the response.
+Some interesting things to note:
+
+- The message-id is 2 because the first message (not shown) was a login request.
+- The response does not include the response payload (i.e. the application status), this information is not logged.
+- The request includes a "patterns" parameter that is set to null, this is specific to the `status` call.
+- The facade information is only included on the request.
+
+.. code:: yaml
+
+    - time: 2023-09-06T10:21:16.72Z
+    conversation-id: "b501bba5508367e5"
+    message-id: 2
+    facade-name: Client
+    facade-method: FullStatus
+    facade-version: 6
+    object-id: ""
+    user-tag: user-alice@external
+    model: controller-1/test-model
+    is-response: false
+    params:
+        patterns: null 
+
+    - time: 2023-09-06T10:21:16.797Z
+    conversation-id: "b501bba5508367e5"
+    message-id: 2
+    facade-name: ""
+    facade-method: ""
+    facade-version: 0
+    object-id: ""
+    user-tag: user-alicen@external
+    model: controller-1/test-model
+    is-response: true
+    params: {}
+    errors:
+        results:
+        - error:
+            code: ""
+            message: ""
+
+Pagination
+~~~~~~~~~~
+
+``--offset``
+    offset the set of returned audit events
+``--limit``
+    limit the maximum number of returned audit events
+
+Audit logs are returned in a paginated set, by default, of size 50. 
+The size of each page can be increased to a maximum of 1000 using the
+``--limit`` flag.
+
+Paging through the result set is also possible with the ``--offset`` flag.
+
+Get second page::
+
+    jimmctl audit-events --offset 50
+
+Change the page size to 100 and get the third page::
+
+    jimmctl audit-events --offset <(page_number-1)*100> --limit 100
+
+Time filters
+~~~~~~~~~~~~
+
+``--after``
+    display events that happened after specified time
+``--before``
+    display events that happened before specified time
+
+Note that all time values must be formatted according to RFC 3339 e.g ``2023-01-02T15:04:05Z``.
+Where the trailing ``Z`` indicates UTC time. To provide a timezone offset
+one can do the following ``2023-12-19T16:39:57-08:00``
+
+Examples::
+
+    jimmctl audit-events --before 2023-10-12T07:20:50.52Z
+    jimmctl audit-events --before 2023-10-12T07:20:50.52-08:00
+    Logs within 1 day
+    jimmctl audit-events --before 2023-10-12T07:20:50.52Z --after 2023-10-11T07:20:50.52Z
+
+Method filter
+~~~~~~~~~~~~~
+
+``--method``
+    display events for a specific method call
+
+Each Juju/Jimmctl call invokes a specific method. This can be thought of as an HTTP handler.
```

Comment:
I think this is very much an implementation detail

Reply:
Should we remove the filter then?